client/src/app/containers/event-card/EventCard.js

```diff
@@ -89,20 +88,10 @@ function EventCard(props) {
               <Moment format='h:mma'>{props.end_date}</Moment>
             </div>
           </p>
-          <button className='popup-button-event-card'>
-            <a
-              href={props.link}
-              target='_blank'
-              rel='noopener noreferrer'
-              className='popup-button-event-card-link'
-            >
-              <span>View on Facebook</span>
-            </a>
-          </button>
         </div>
         <div className='desc-text-wrapper-event-card'>
           <p className='desc-text-event-card'>
-            <Linkify> {props.description}</Linkify>
+            <div dangerouslySetInnerHTML={{ __html: props.description }} />
```

Comment:
Question: was there a reason for using `dangerouslySetInnerHTML` instead of Linkify?

Reply:
Google Calendar provides the event description as HTML tags from their available formatting options (bold, italic, underline, unordered list, ordered list, and links). react-linkify was only for turning link text into anchors and cannot parse other HTML elements.